examples/deck/example.js

```diff
@@ -96,15 +146,14 @@ export default class Example extends Component<
       width: window.innerWidth
     };
 
-    const editablePolygonsLayer = new EditablePolygonsLayer({
+    const editableGeoJsonLayer = new EditableGeoJsonLayer({
       data: testFeatures,
       selectedFeatureIndex,
-      pickable: true,
-      isEditing: true,
+      isEditing: this.state.isEditing,
```

Comment:
What is this flag? Seems too stateful, not declarative... Just call it `editable`?